### ISSUE TITLE
Consistent formatting of MOM_input, matching MOM_parameter_doc.short

### ISSUE THREAD

**Issue by aekiss** [ACCESS-NRI/access-om3-configs#382]
We need a consistent `MOM_input` format so we can easily compare across branches using `git diff` and also avoid merge conflicts when cherry-picking across branches.

I propose we use an order and format that matches `MOM_parameter_doc.short` so it will also be easy to see whether anything in `MOM_input` is redundant (e.g. a default value) by comparing `MOM_input` to `MOM_parameter_doc.short`. Note that this differs from the previous layout we inherited from CESM - e.g. see [here](https://github.com/COSIMA/MOM6-CICE6/commit/7dd28b19b506daf313eeefd2346e5ae7e151ab2b#diff-bf0915852240640bb6bc6b27a0d786446acb8f242710b1757994086c2e8b91ba). I've already done this for [`MOM6-CICE6/1deg_jra55do_ryf`](https://github.com/COSIMA/MOM6-CICE6/blob/1deg_jra55do_ryf/MOM_input) and propose doing this for our other branches.

Also note that there will need to be some hand-additions of the parameters not included in `MOM_parameter_doc.short` - see https://github.com/COSIMA/access-om3/issues/121

Any objections?

**Comment by anton-seaice** [comment]
@ACCESS-NRI/ocean - do we think it's worth trying to automate this (probably add CI which fails if the new MOM_input doesn't match `MOM_parameter_doc.short`) ? Or just leave it as something for reviewers to confirm ?

**Comment by dougiesquire** [comment]
I think leaving to the reviewer is fine for now? It's not really the end of the world if things get out of sync and then get realigned in a later PR

**Comment by aekiss** [comment]
agreed - maybe have a PR checkbox to to remind the committer to `diff MOM_parameter_doc.short MOM_input`?

**Comment by aekiss** [comment]
that would help avoid having to fix multiple branches if a mis-formatted MOM_input is widely cherry-picked

**Comment by anton-seaice** [comment]
It's in the pull_request_template:

https://github.com/ACCESS-NRI/access-om3-configs/blob/ad8788365f758cba6cfb48adf7ec78f81dee9018/.github/pull_request_template.md?plain=1#L24

Maybe re-word to 

`Are changes to MOM_input in the same order as docs/MOM_parameter_doc.short?`

**Comment by dougiesquire** [comment]
> Maybe re-word to
> 
> `Are changes to MOM_input in the same order as docs/MOM_parameter_doc.short?`

Maybe make it a checkbox as well? Otherwise it's not clear whether someone has done it or not. Many of the others could also be checkboxes

**Comment by aekiss** [comment]
would it be helpful to require the same formatting, comments, whitespace, etc? ie requiring a check with `diff MOM_parameter_doc.short MOM_input`? 

Probably only worthwhile if we're often comparing `MOM-input` and `MOM_parameter_doc.*` and want to minimise irrelevant clutter?

**Comment by anton-seaice** [comment]
> would it be helpful to require the same formatting, comments, whitespace, etc? ie requiring a check with `diff MOM_parameter_doc.short MOM_input`?

Oh consistency of all that was implied to me, so I will update the checklist to be clearer

**Comment by anton-seaice** [comment]
The change has been made in the main configuration branches, and also added to the pull request template.
